patches/minecraft/net/minecraft/client/renderer/BufferBuilder.java.patch

```diff
@@ -58,5 +58,13 @@
 +    public boolean isColorDisabled()
 +    {
 +        return this.field_78939_q;
++    }
++
++    public void addVertexData(ByteBuffer buffer)
++    {
++        func_181670_b(buffer.limit());
```

Comment:
This does have the same issue, you need to change this to 'func_181670_b(buffer.limit() + this.field_179011_q.func_177338_f()'